docs/src/backup_recovery.md

```diff
@@ -358,11 +384,26 @@ spec:
 ```
 
 This, will tell the operator that the cluster is running inside a Google Kubernetes
-Engine meaning that no credentials are needed to upload the files
+Engine meaning that no credentials are needed to upload the files.
 
-!!! Important
-    This method will require carefully defined permissions for cluster
-    and pods, which have to be defined by a cluster administrator.
+However this will require proper [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) configuration and a specifica `annotation` in the cluster's service account.
```

Comment:
```suggestion
However this will require proper [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) configuration and a specific `annotation` in the cluster's service account.
```